### DISCUSSION TITLE
Getting kicked for "added buff to npc abnormaly"

### DISCUSSION THREAD

**Discussion by J-Zep** [Pryaxis/TShock#2133] [Q&A]
Hey there,
since the new update is there i get kicked for added buff to npc abnormaly, when i'm using my Fire Gauntlet and hitting an npc. Does someone know what i'm doing wrong or what i'm missing to add for a permission? Ty

**Comment by Quinci135** [comment]
Sounds like you haven't updated properly, dId you replace OTAPI.dll and TerrariaServer.exe with the ones from the newest pre-release and replace your TShockAPI.dll in your ServerPlugins folder? You can see if you are running an older version of TShock if you are unable to spawn in any 1.4.1 items or mobs.
